py-polars/tests/unit/namespaces/test_meta.py

```diff
@@ -71,7 +71,7 @@ def test_meta_is_regex_projection() -> None:
 
 
 def test_meta_tree_format() -> None:
-    with Path("tests/unit/namespaces/test_tree_fmt.txt").open(
+    with Path(__file__).parent.joinpath("test_tree_fmt.txt").open(
```

Comment:
Well-spotted 😉  I saw the same thing and committed a PR to address this a bit more generally for the entire `namespace` tests dir; you can revert this for now as https://github.com/pola-rs/polars/pull/11588 will handle it 👍

Reply:
merged with main, so this is outdated now